build.gradle

```diff
@@ -428,7 +432,7 @@ subprojects {
         sign publishing.publications.maven
     }
 
-    [publishMavenPublicationToMavenRepository, publishMavenPublicationToMavenLocal]*.onlyIf {
```

Comment:
Go ahead and leave this. I find it unexpected when maven local installs produce different results.

Reply:
done